src/passwordValidation.js

```diff
@@ -21,6 +19,14 @@ export default function isValidPassword(password = "") {
   let majiscule = /^[A-Z0-9]+$/;
   let miniscule = /^[a-z0-9]+$/;
   let forbPass = forbiddenPasswords.includes(password);
+  let response = false;
+  let regexNumbered = /\d+/g;
+  let Numbered = password.match(regexNumbered).join("");
```

Comment:
Also here avoid  to the create  unnecessary variables instead pass the values directly where they are being used
```suggestion
  let Numbered = password.match(/\d+/g).join("");
```